examples/kuesa/shared-utils/shared-utils.pri

```diff
@@ -0,0 +1,29 @@
+# shared-utils.pri
+#
+# This file is part of Kuesa.
+#
+# Copyright (C) 2018-2019 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+# Author: Mike Krus <mike.krus@kdab.com>
+#
+# Licensees holding valid proprietary KDAB Kuesa licenses may use this file in
+# accordance with the Kuesa Enterprise License Agreement provided with the Software in the
+# LICENSE.KUESA.ENTERPRISE file.
+#
+# Contact info@kdab.com if any conditions of this licensing are not clear to you.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+LIBS += -L../shared-utils -lshared-utils
+win32:CONFIG(release, debug|release): LIBS += -L../shared-utils/release/
```

Comment:
why not have it as
win32: {
CONFIG(release, debug|release): LIBS += -L../shared-utils/release/
CONFIG(debug, debug|release): LIBS += -L../shared-utils/debug/
}

Reply:
good call